__tests__/mixin.test.ts

```diff
@@ -1,107 +1,107 @@
-import { config, mount } from '@vue/test-utils';
-import { fpjsGetVisitorDataExtendedMixin, fpjsGetVisitorDataMixin, fpjsPlugin, FpjsVueOptions } from '../src';
-import { getVisitorData } from 'shared/tests/setup';
-import { wait } from 'shared/utils';
+import { config, mount } from '@vue/test-utils'
+import { fpjsGetVisitorDataExtendedMixin, fpjsGetVisitorDataMixin, fpjsPlugin, FpjsVueOptions } from '../src'
```

Comment:
We could keep prettier semicolon rule to not cause such code changes

Reply:
It makes sense, but I think it's better to have consistent formatting in all of our repos. WDYT?

Reply:
I just don't want to destroy git blame with bulk change. But as you wish.